### PR TITLE
initial pass at addressing feedback from John Klensin

### PR DESCRIPTION
**Pull request by stpeter**
Fixes https://github.com/richsalz/draft-ietf-uta-rfc6125bis/issues/83

### CODE REVIEW

**Review by richsalz — APPROVED** [review]
LGTM.  A few suggestions, do or not do as Yoda would say ;)

File: draft-ietf-uta-rfc6125bis.md [hunk 1]
```diff
@@ -461,21 +482,23 @@ server at example.net") or a URI.
...
-
-An IP address is either a 4-byte IPv4 address {{!IPv4=RFC0791}} or a 16-byte
+   least one label containing appropriately encoded Unicode code points
+   outside the traditional US-ASCII range and conforming to the processing
+   and validity checks specified for "IDNA2008" in {{IDNA-DEFS}} and the 
+   associated documents. In particular, it contains at least one U-label or 
+   A-label, but otherwise may contain any mixture of NR-LDH labels, A-labels, 
+   or U-labels.
+
+An IP address is either a 4-octet IPv4 address {{!IPv4=RFC0791}} or a 16-octet
```
[the remark this answers is not in the document]
Comment (stpeter) [reply draft-ietf-uta-rfc6125bis.md:501]: I suppose I'm agnostic about that, but we should make it clear. This is text that @martinthomson contributed so perhaps he had an opinion in the matter.

[the remark this answers is not in the document]
Comment (martinthomson) [reply draft-ietf-uta-rfc6125bis.md:501]: The change here is s/byte/octet/ as it is a description of the canonical form (as used in certificates).  Presumably the textual representation is parsed out.

File: draft-ietf-uta-rfc6125bis.md [hunk 2]
```diff
@@ -461,21 +482,23 @@ server at example.net") or a URI.
...
 The DNS name conforms to one of the following forms:
 
-1. A "traditional domain name", i.e., a FQDN (see {{DNS-CONCEPTS}}) all of
-  whose labels are "LDH labels" as described in {{IDNA-DEFS}}.  Informally,
-  such labels are constrained to {{US-ASCII}} letters, digits, and the
-  hyphen, with the hyphen prohibited in the first character position.
-  Additional qualifications apply (refer to the above-referenced
-  specifications for details), but they are not relevant here.
+1. A "traditional domain name", i.e., a FQDN that conforms to "preferred
+   name syntax" as described in Section 3.5 of {{DNS-CONCEPTS}} and for which all of
```
Comment (martinthomson) [inline draft-ietf-uta-rfc6125bis.md:486]:
```suggestion
   name syntax" as described in {{Section 3.5 of DNS-CONCEPTS}} and for which all of
```

File: draft-ietf-uta-rfc6125bis.md [hunk 3]
```diff
@@ -577,31 +600,41 @@ Note that some of these rules are cumulative
...
-4. The certificate MAY contain more than one DNS-ID, SRV-ID, URI-ID, or IP-ID
-  as further explained under {{security-multi}}.
-
-5. The certificate MAY include other application-specific identifiers
-  for compatibility with a deployed base. Such identifiers are out of
-  scope for this specification.
+   the relevant specification stipulates that certificates should
+   include identifiers of type "SRV-ID" (e.g., this is true of {{XMPP}}),
+   then the certificate SHOULD include an SRV-ID.  Typically this
+   identifier type would supplement the DNS-ID, unless the certificate
```
Comment (martinthomson) [inline draft-ietf-uta-rfc6125bis.md:614]:
```suggestion
   then the certificate SHOULD include an SRV-ID.  This
   identifier type could supplement the DNS-ID, unless the certificate 
```

**Review by martinthomson — COMMENTED** [review]
That's a lot of new text.

File: draft-ietf-uta-rfc6125bis.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (stpeter) [reply draft-ietf-uta-rfc6125bis.md:501]: True. So I think we're good on this score.

**Pull request merged**